apps/common/src/main/scala/org/lfdecentralizedtrust/splice/store/StoreMetrics.scala

```diff
@@ -36,6 +36,34 @@ class StoreMetrics(metricsFactory: LabeledMetricsFactory)(metricsContext: Metric
       0L,
     )(metricsContext)
 
+  val lastIngestedOffset: Gauge[Long] = metricsFactory.gauge(
+    MetricInfo(
+      name = prefix :+ "last-ingested-offset",
+      summary = "The last offset ingested by this store",
+      Latency,
+      "The latest offset ingested by this store in numeric form.",
+    ),
+    0L,
+  )(metricsContext)
+
+  val ingestedTxLogEntries: Counter = metricsFactory.counter(
```

Comment:
let prefer meters instead of counters (meters only go up, counters go up and down)